ovssubnet/registry/registry.go

```diff
@@ -142,28 +132,35 @@ func (sub *EtcdSubnetRegistry) InitNodes() error {
 	return err
 }
 
-func (sub *EtcdSubnetRegistry) GetNodes() (*[]string, error) {
+func (sub *EtcdSubnetRegistry) GetNodes() (*[]string, *[]string, error) {
```

Comment:
Let's return node objects (create new if needed). Two arrays of strings seems odd.